### ISSUE TITLE
Server login/config help required

### ISSUE THREAD

**Issue by dawiepoolman** [ShahanaFarooqui/RTL#92]
Hi guys

I have installed nodejs & npm on my RPi and I have this RTL.conf on my RPi:
 
![image](https://user-images.githubusercontent.com/2351212/54809859-2a94b800-4c8d-11e9-9b20-3dd31b258a36.png)

I have enabled port 3000 on my RPi firewall and my node rtl seems to be up and running:
![image](https://user-images.githubusercontent.com/2351212/54809987-852e1400-4c8d-11e9-95f8-4955e4da0a77.png)

When I browse from my laptop to my RPi at http://192.168.1.20:3000 the url won't connect:
![image](https://user-images.githubusercontent.com/2351212/54810468-e1ddfe80-4c8e-11e9-8949-c2c53a2bc715.png)

How can I debug the connection issue?  
Pls ELI5, I am quite green when it comes to LND & Linux but willing to learn
TIA

**Comment by dawiepoolman** [comment]
cont../

When adding RTL as a service on my headless RPi I get his wen I start the service:
![image](https://user-images.githubusercontent.com/2351212/54814829-2d95a580-4c99-11e9-83d6-ec57f9ad9184.png)

The contents of /etc/systemd/system/RTL.service
![image](https://user-images.githubusercontent.com/2351212/54814964-7f3e3000-4c99-11e9-9b5c-23091c7445c7.png)

**Comment by dawiepoolman** [comment]
nvm, got the RTL.service  running
I had the node dir incorrect.  I have it installed at:
ExecStart=/usr/**local**/bin/node /home/admin/RTL/rtl 

However, my laptop browser still does not connect

**Comment by saubyk** [comment]
@dawiepoolman 
Can you please share the ‘ufw status’? Just share the one for port 3000 only.

Is there any firewall rule, which you need to set for your laptop browser?

**Comment by dawiepoolman** [comment]
@saubyk 
ty for the response
ufw status for port 3000: 
![image](https://user-images.githubusercontent.com/2351212/54820994-984edd00-4ca9-11e9-9ff2-419d2ccf6208.png)

my laptop LAN IP is 192.168.1.22 which is in the allowed range 

you might have a point on the router firewall?
do I need to port fwd explicitly? I can ping and SSH to the RPi tough

**Comment by dawiepoolman** [comment]
fudge.. I think I see it.. facepalm.. brb

**Comment by dawiepoolman** [comment]
lulz, I had my IP incorrect in my ufw
it works now thank you!
